app/src/main/java/org/eyeseetea/malariacare/EyeSeeTeaApplication.java

```diff
@@ -59,6 +60,9 @@
     public void onCreate() {
         super.onCreate();
         Log.d(TAG, "onCreate");
+        if(BuildConfig.elementSDK) {
```

Comment:
This init shouldn't happen directly here (see the dependency that's created via the import with the ElementSDKManager). Instead of that, I would rely on the ElementController for that purpose. The better would be something like a plugin initialization where all the Controllers that need to be initialized simply subscribe and are initialized here, but if that's too much, at least it's better to have a dependency with our generic controller and not with the element SDK.